kratos/solving_strategies/schemes/residual_based_bdf_custom_scheme.h

```diff
@@ -621,37 +621,78 @@ class ResidualBasedBDFCustomScheme
         const std::size_t n_second_derivative = ThisParameters["second_derivative"].size();
 
         // Size check
-        KRATOS_ERROR_IF(n_variables != n_first_derivative) << "Your list of variables is not the same size as the list of first derivatives variables" << std::endl;
-        KRATOS_ERROR_IF(n_variables != n_second_derivative) << "Your list of variables is not the same size as the list of second derivatives variables" << std::endl;
+        KRATOS_ERROR_IF(n_variables != n_first_derivative && n_first_derivative > 0) << "Your list of variables is not the same size as the list of first derivatives variables" << std::endl;
+        KRATOS_ERROR_IF(n_variables != n_second_derivative && n_second_derivative > 0) << "Your list of variables is not the same size as the list of second derivatives variables" << std::endl;
+        KRATOS_WARNING_IF("ResidualBasedBDFCustomScheme", n_first_derivative == 0) << "Your first derivatives variables is empty, it will be determined automatically using VariablesDerivatives" << std::endl;
+        KRATOS_WARNING_IF("ResidualBasedBDFCustomScheme", n_second_derivative == 0) << "Your ssecond derivatives variables is empty, it will be determined automatically using VariablesDerivatives" << std::endl;
```

Comment:
With this PR it is no longer necessary to give the derivatives in the input
@KratosMultiphysics/technical-committee explicitly does not want this to be variable

one variable always has the same derivative

Reply:
Yes, I saw later

Reply:
On it